Jenkinsfile_CNP

```diff
@@ -26,7 +26,16 @@ def secrets = [
     secret('payments-queue-send-connection-string', 'PAYMENTS_QUEUE_WRITE_CONN_STRING')
   ]
 ]
-
+def prSecrets = [
+  's2s-${env}': [
+    secret('microservicekey-bulk-scan-orchestrator', 'S2S_SECRET')
```

Comment:
you can copy  the secret in bulk scan vault and load it from there ?

Reply:
we need to use osba created queue in PR

Reply:
not sure how osba and s2s are related

Reply:
got you now